src/content/app/tools/blast/types/blastSettings.ts

```diff
@@ -85,13 +85,24 @@ type ProgramConfiguration = {
   programs: BlastProgram[];
 };
 
+export type ParameterRange = {
+  [program: string]: {
+    min: number;
+    max: number;
+  };
+};
+type ProgramParametersLimit = {
```

Comment:
If we go with the min/max approach, we should review the use of the word "limit". I am not sure that the word _limit_ (singular) can mean both min and max at the same time.